### DISCUSSION TITLE
Translation into Hungarian

### DISCUSSION THREAD

**Discussion by thepali1** [aclap-dev/video-downloadhelper#1169] [Language & Translation]
Hi!

I have attached the Hungarian translation: [hu.txt](https://github.com/user-attachments/files/16681941/hu.txt)
I will gladly accept a free license. :)

**Comment by P-Chris75** [comment]
[hu.txt](https://github.com/user-attachments/files/16914053/hu.txt)

I am also attaching here the full Hungarian translation, can be compared with the proposals of others.
